app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb

```diff
@@ -0,0 +1,279 @@
+module Hyrax
+  module Dashboard
+    module CollectionsControllerBehavior
```

Comment:
My inclination is to not make the refactor even more complex.  I will ticket this for follow-on work, given the codebase does not consistently adopt the new behavior-less approach.